src/argilla/client/feedback/dataset/local.py

```diff
@@ -230,3 +230,45 @@ def add_metadata_property(
         self._metadata_properties.append(metadata_property)
         self._metadata_properties_mapping.update({metadata_property.name: metadata_property})
         return metadata_property
+
+    def delete_metadata_properties(
```

Comment:
metadata methods should be aligned. IF there is a method `add_metadata_property` should be also the equivalent for deletion `delete_metadata_property` 

Also, all common methods should be defined in the FeedbackDataset base class. This will assure the Remote/Local API alignment